__tests__/database.ts

```diff
@@ -1,7 +1,52 @@
-import { deleteSession, getSession, insertSession, updateSession } from "@/data/botData";
+import { createModuleConfigRecord, deleteModuleConfigRecord, deleteSession, getModuleConfigRecord, getSession, insertSession, updateModuleConfigRecord, updateSession } from "@/data/botData";
 import { describe } from "node:test";
 
 describe('Database', () => {
+    // module_config
+    test('can create module_config record', async () => {
+        return createModuleConfigRecord({
+            serverId: 1,
+            moduleId: 1,
+            moduleConfig: "{}",
+            enabled: true
+        })
+        .then(response => expect(response).toEqual(true))
+    })
+
+    test('can update a module_config record', async () => {
+        await createModuleConfigRecord({
+            serverId: 1,
+            moduleId: 1,
+            moduleConfig: "{}",
+            enabled: true
+        })
+
+        const success = await updateModuleConfigRecord({
+            serverId: 1, 
+            moduleId: 1, 
+            moduleConfig: "{}", 
+            enabled: false
+        })
+        
+        if (!success)
+            return false
+
+        return getModuleConfigRecord(1, 1)
+        .then(response => response?.enabled)
+        .then(enabled => expect(enabled).toEqual(false))
```

Comment:
Why not just access response?.enabled directly?
.then(enabled => expect(response?.enabled).toBe(false))